go.mod

```diff
@@ -19,10 +19,10 @@
 
 module github.com/apache/incubator-yunikorn-scheduler-interface
 
-go 1.12
+go 1.15
```

Comment:
I think for updating the go version is better to create a different Jira, it makes easyer to trach the changes, since this issue is a bout a build failure, not about go version upgrade.

Reply:
Sure, will address this in a follow-up JIRA